### ISSUE TITLE
Please resume maintenance or give project to new maintainer

### ISSUE THREAD

**Issue by flying-sheep** [compulim/vscode-mocha#18]
There’s a very important bug unfixed even though a simple PR fixing it exists: #7

Please allow this project to go forward by resuming maintenance or giving access to it to someone who does.

**Comment by dsole** [comment]
I have moved to mocha-latte. Maybe do the same ?

> This is a maintained, bugfixed fork of the Mocha extension. Unlike the original, it also works on Mac and Linux. To use "Mocha Latte", uninstall the "Mocha" extension.

However I agree - maintaining this original repo ( or removing it from the marketplace to avoid confusion ) would be better

https://marketplace.visualstudio.com/items?itemName=cspotcode.vscode-mocha-latte
https://github.com/cspotcode/vscode-mocha-latte

**Comment by flying-sheep** [comment]
you’re right, i meant that they should be merged under the more basic name

besides, mocha latte is also not very active
